### PR TITLE
fix: fix acl file parsing

### PR DESCRIPTION
**Pull request by shyimo**
- Fix Redis ACL plain test parsing error
- Handle ACL specific commands declaration error

### CODE REVIEW

File: controllers/rediscli/acl.go [hunk 1]
```diff
@@ -78,6 +75,7 @@ func parseACL(rawData string) (*RedisACL, error) {
 	rawData = strings.Trim(rawData, "\n")
 	entries := strings.Split(rawData, "\n")
 	for i, e := range entries {
+		fmt.Printf("ACL entrie: %s\n", e)
```
Comment (voltbit) [inline controllers/rediscli/acl.go:78]: Lets remove this printf, will probably make too much noise.

File: controllers/config_controller.go [hunk 2]
```diff
@@ -56,6 +56,7 @@ const ACLFilePropagationDuration time.Duration = time.Millisecond * 5000
 // Defines the time it takes for Redis to load the new config
 const ACLFileLoadDuration time.Duration = time.Millisecond * 500
 const redisConfigLabelKey string = "redis-cluster"
+const handleACLConfigErrorMessage = "Failed to handle acl configuation"
```
Comment (voltbit) [inline controllers/config_controller.go:59]: Nitpick: ACL instead of acl

**Review by voltbit — APPROVED** [review]
looks good 👍

File: controllers/config_controller.go (hunk 2 above)
Comment (shyimo) [reply controllers/config_controller.go:59]: done

File: controllers/rediscli/acl.go (hunk 1 above)
Comment (shyimo) [reply controllers/rediscli/acl.go:78]: 👍

**Pull request merged**